simple-generated/Cargo.toml

```diff
@@ -6,4 +6,6 @@ license = "MIT"
 edition = "2021"
 
 [dependencies]
+crossterm = "0.28.1"
```

Comment:
I think this is better to leave in based on seeing some of the comments in the issue related to this.